### DISCUSSION TITLE
新版本如何在chrome中调试

### DISCUSSION THREAD

**Discussion by 34892002** [midwayjs/midway#4122] [General / 闲聊]
老版本使用 cross-env NODE_ENV=local midway-bin dev --ts --debug 可以开启，新版本还是一样吗？

**Comment by 34892002** [comment]
自问自答一波： npm i @midwayjs/cli -D 之后就可以用了
另外问一下，@midwayjs/cli后续是不会更新了吗？那之后怎么使用浏览器调试？毕竟浏览器控制台用来查看输出数据还是很方便的

**Reply by czy88840616** [reply]
@midwayjs/cli 里面过度复杂，肯定不会更新了。浏览器调试其实只要打开 chrome://inspect，本地用 inspect 模式启动就可以看到了。。

**Comment by czy88840616** [comment]
mwtsc 新版本加好了。

<img width="1146" alt="image" src="https://github.com/user-attachments/assets/a568b3a3-7956-4f81-b2c1-de7fb3e028c7">

启动增加 `--inspect` 以及需要配置 inlineSource 和 inlineSourcemap

```
"dev": "cross-env NODE_ENV=local mwtsc --watch --inspect --run @midwayjs/mock/app.js",
```

tsconfig.json
```json
{
  "compileOnSave": true,
  "compilerOptions": {
...
    "inlineSourceMap":true,
    "inlineSources": true,
  },
}
```

**Comment by 34892002** [comment]
厉害了，谢谢大佬，疯狂点赞👍👍👍👍

**Comment by 34892002** [comment]
大佬，还一个小小的问题，
每次修改代码保存之后，调试器就自动断开了必须重新连接，有办法配置吗？
Waiting for the debugger to disconnect...

**Reply by czy88840616** [reply]
不能的，服务每次重启会生成一个新的 guid websocket，必须重连的
